<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>controller</class>
 <widget class="QWidget" name="controller">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>controller</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>414</width>
     <height>511</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="3">
     <widget class="QPushButton" name="btnF3">
      <property name="text">
       <string>功能1</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="QProgressBar" name="progressBar1">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="4">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>后台信息</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="btnF1">
      <property name="text">
       <string>功能1</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="btnF4">
      <property name="text">
       <string>功能2</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2" rowspan="2">
     <widget class="QPushButton" name="btnTerm2">
      <property name="text">
       <string>正常终止</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" rowspan="2">
     <widget class="QProgressBar" name="progressBar2">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0" rowspan="2">
     <widget class="QPushButton" name="btnStart2">
      <property name="text">
       <string>线程2开始</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="btnF2">
      <property name="text">
       <string>功能2</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" rowspan="2">
     <widget class="QPushButton" name="btnTerm1">
      <property name="text">
       <string>异常终止</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="2">
     <widget class="QPushButton" name="btnStart1">
      <property name="text">
       <string>线程1开始</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="4">
     <widget class="QPlainTextEdit" name="plainTextEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
